.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.3.3
```

Comment:
... tried, and a number of gems (with no available upgrade) warn of `warning: constant ::Fixnum is deprecated` amongst other things.

It's probably harmless, but does make for a bigger change, so... rain check?